extensions/wikia/WikiaMobile/WikiaMobileCategoryService.class.php

```diff
@@ -72,13 +72,14 @@ public function alphabeticalList() {
 		 * @var $categoryPage CategoryPage
 		 */
 		$categoryPage = $this->request->getVal( 'categoryPage' );
+		$isMercury = $this->request->getBool( 'isMercury' );
```

Comment:
I'm not totally convinced about this isMercury flag however I couldn't think of anything better so if someone has a better idea I would be very happy!

*What issue this flag solves*
Inside BlogArticle there is check if skin is set to wikiamobile - in case it is the blog posts in category should be nicely formatted. We want the same thing to happen in case of mercury however in case of mercury we don't have skin set in MW.